### PR TITLE
Fix Cumulative Distribution Plot

### PR DESCRIPTION
**Pull request by kassiwu**
### Changes
- Fix background width
- Use the data set points as the X axis and cumulative percentages as the Y Axis (scatter plot)
- Force to start Y-axis always with zero 
- Use transform property to find the tick on the top position of y axis and make sure the tick is not cut off in the downloaded plot

### Demo
chart - 2019-04-15T160526 661

### CODE REVIEW

File: interface/components/cumulative-distribution/index.coffee [hunk 1]
```diff
@@ -127,12 +132,26 @@ ko.components.register "tf-cumulative-distribution",
         e.style.fill = "none"
 
       svg_element.style.backgroundColor = "white"
+      tick = svg_element.querySelectorAll ".tick"
+      num_arr = Array(tick.length).fill(0).map((x, y) => y)
+
+      for num in num_arr
+        # use transform property to check if the SVG element is on the top position of y axis
+        transform_y_val = (getComputedStyle(tick[num]).getPropertyValue('transform').replace(/^matrix(3d)?\((.*)\)$/,'$2').split(/, /)[5])*1
```
Comment (tsangjustin) [inline interface/components/cumulative-distribution/index.coffee]: Can you add what the regex here is looking for and doing

File: interface/components/cumulative-distribution/index.coffee [hunk 2]
```diff
@@ -164,6 +183,18 @@ ko.components.register "tf-cumulative-distribution",
     @column_index.subscribe ( next ) =>
       if next then adapter.unsubscribeToChanges()
       else adapter.subscribeToChanges()
+      if @active()
+        if (@values().length)
```
Comment (tsangjustin) [inline interface/components/cumulative-distribution/index.coffee]: you can put this into one if condition instead of nested if conditions

**Comment by tsangjustin** [comment]
@kassiwu @a311916000 Why does this PR have similar code as the histogram PR? Are they the same changes?

**Comment by kassiwu** [comment]
No, I made different changes to the two branches. We now do not need to sort values into "buckets" and have scatter plot for cumulative distribution.

**Pull request merged**